Squeezer/src/main/java/uk/org/ngo/squeezer/service/SqueezeService.java

```diff
@@ -796,9 +792,14 @@ private void handleRandomOnEvent(Player player) {
                 try {
                     RandomPlayDelegate.fillPlaylist(unplayed, player, next);
                 } catch (Exception e) {
-                    Log.e(TAG, "handleRandomOnEvent: (RandomPlay) Unable to fill playlist");
+                    Log.e(TAG, "handleRandomOnEvent: Unable to fill playlist");
```

Comment:
Remove this unnecessary try/catch block.

Reply:
Removed the try/catch.